runtime_lib/xaiengine/CMakeLists.txt

```diff
@@ -44,7 +44,11 @@ if (${CMAKE_CROSSCOMPILING})
 
 else()
     message("Copying xaiengine for ${AIE_RUNTIME_TARGET} from Vitis at ${VITIS_ROOT}.") 
-    set(xaiengineIncludePath ${VITIS_ROOT}/aietools/include/drivers/aiengine)
+    set(xaiengineIncludePath ${VITIS_AIETOOLS_DIR}/include/drivers/aiengine)
+    #tmp path to header files since we use an older (3_0) version of xaiengine. need to fix when we upgrade to 2023.1 and can take this from aie-rt
+    set(xaiengineIncludePathTmp ${VITIS_ROOT}/data/embeddedsw/XilinxProcessorIPLib/drivers/aienginev2_v3_0/src)
```

Comment:
I'd rather not proliferate this hard-coded path. We already have a problem that this version isn't consistent with the version used in aiesimulator.

Reply:
made path dependent on Vitis Version